src/reporters/lang/en.js

```diff
@@ -114,6 +114,8 @@ const messages = {
   allDependenciesUpToDate: 'All of your dependencies are up to date.',
   legendColorsForUpgradeInteractive:
     'Color legend : \n $0    : Major Update backward-incompatible updates \n $1 : Minor Update backward-compatible features \n $2  : Patch Update backward-compatible bug fixes',
+  legendColorsForOutdated:
```

Comment:
This is the same as `legendColorsForUpgradeInteractive` should I merge them? If yes what to name them?

Reply:
yeah I'd prefer merging them. How about `legendColorsForVersionUpdates`?

Reply:
Okay changed :)